<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件系统Model/View</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>TreeView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>ListView</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="LabFileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabFileSize">
           <property name="text">
            <string>文件大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabType">
           <property name="text">
            <string>节点类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkIsDir">
           <property name="text">
            <string>节点是目录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="LabPath">
         <property name="text">
          <string>路径名：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(E)</string>
    </property>
    <addaction name="actionUndo_U_Ctrl_Z"/>
    <addaction name="separator"/>
    <addaction name="actionCut_T_Ctrl_X"/>
    <addaction name="actionCopy_C_Ctrl_C"/>
    <addaction name="actionPaste_P_Ctrl_V"/>
    <addaction name="actionDelete_L_Del"/>
    <addaction name="separator"/>
    <addaction name="actionFind_F_Ctrl_F"/>
    <addaction name="actionFind_Next_N_F3"/>
    <addaction name="actionReplace_R_Ctrl_H"/>
    <addaction name="actionGoto_G_Ctrl_G"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All_A_Ctrl_A"/>
    <addaction name="actionTime_Date_D_F5"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New(N)                Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open(O)               Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save(S)                 Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save As(A)        </string>
   </property>
  </action>
  <action name="actionUndo_U_Ctrl_Z">
   <property name="text">
    <string>Undo(U)               Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut_T_Ctrl_X">
   <property name="text">
    <string>Cut(T)                  Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_C_Ctrl_C">
   <property name="text">
    <string>Copy(C)               Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_P_Ctrl_V">
   <property name="text">
    <string>Paste(P)               Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete_L_Del">
   <property name="text">
    <string>Delete(L)                  Del</string>
   </property>
  </action>
  <action name="actionFind_F_Ctrl_F">
   <property name="text">
    <string>Find(F)...               Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next_N_F3">
   <property name="text">
    <string>Find Next(N)...            F3</string>
   </property>
  </action>
  <action name="actionReplace_R_Ctrl_H">
   <property name="text">
    <string>Replace(R)...        Ctrl+H</string>
   </property>
  </action>
  <action name="actionGoto_G_Ctrl_G">
   <property name="text">
    <string>Goto(G)...            Ctrl+G</string>
   </property>
  </action>
  <action name="actionSelect_All_A_Ctrl_A">
   <property name="text">
    <string>Select All(A)         Ctrl+A</string>
   </property>
  </action>
  <action name="actionTime_Date_D_F5">
   <property name="text">
    <string>Time/Date(D)             F5</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
